inst/include/cpp11/integers.hpp

```diff
@@ -144,25 +144,32 @@ inline int na() {
   return NA_INTEGER;
 }
 
-// forward declaration
-
+// forward declarations
 typedef r_vector<double> doubles;
 
+template <>
+bool is_na(const double& x);
```

Comment:
This time there _is_ an `is_na()` specialization for doubles (unlike for integers), so we forward declare that here instead of `na<double>()`.